.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 node-image: &node-image
-  image: circleci/node:15
+  image: circleci/node:16
```

Comment:
This could make sense 🤷

Reply:
right that's what I meant :D I thought I put the comment here :)